docs/MiddlewareAPICommands.md

```diff
@@ -0,0 +1,109 @@
+# Middleware APIs Commands
+
+## Identity context
+
+### AccreditIdentityCommand
+- RA command
+
+|**Fieldname**|**Required**|**Type**|**Remarks**|
+|--|--|--|--|
+|`identityId`|yes|`string`|UUID of the Identity|
+|`institution`|yes|`string`|UUID of the Institution of the Identity|
+|`role`|yes|`string`|The rolename the Identity is accredited with, possible options `[ra, raa]`|
+|`contactInformation`|yes|`string`|Contact information of the Identity|
+|`raInstitution`|yes|`string`|UUID of the RA Institution the Identity is accredited a role at.|
+
+### AddToWhitelistCommand
+- Management command
+
+|**Fieldname**|**Required**|**Type**|**Remarks**|
+|--|--|--|--|
+|`institutionsToBeAdded`|yes|`array` of `string`|A list of Institutions to be added to the whitelist|
+
+### AmendRegistrationAuthorityInformationCommand
+- RA command
+
+|**Fieldname**|**Required**|**Type**|**Remarks**|
+|--|--|--|--|
+|`identityId`|yes|`string`|UUID of the Identity|
+|`contactInformation`|yes|`string`|Contact information of the Identity|
+|`raInstitution`|yes|`string`|UUID of the RA Institution the Identity is amending information at.|
+
+### AppointRoleCommand
+- RA command
+
+|**Fieldname**|**Required**|**Type**|**Remarks**|
+|--|--|--|--|
+|`identityId`|yes|`string`|UUID of the Identity|
+|`role`|yes|`string`|The rolename the Identity is accredited with, possible options `[ra, raa]`|
+|`raInstitution`|yes|`string`|UUID of the RA Institution the Identity is appointed a role at.|
+
+### BootstrapIdentityWithYubikeySecondFactorCommand
+Used to bootstrap SRAA users
+
+- Admin command, fired from command line using Symfony console command
+
+|**Fieldname**|**Required**|**Type**|**Remarks**|
+|--|--|--|--|
+|`identityId`|yes|`string`|UUID of the Identity|
+|`nameId`|yes|`string`|The nameId of the Identity|
```

Comment:
in the response to the `/identity` api-calls this attribute is `name_id`.  We should use only one notation.